src/test/java/redis/clients/jedis/tests/commands/PublishSubscribeCommandsTest.java

```diff
@@ -82,7 +84,7 @@ public void onSubscribe(String channel, int subscribedChannels) {
   }
 
   @Test
-  public void pubSubChannelWithPingPong() throws InterruptedException {
+  public void pubSubChannelWithPingPong() {
```

Comment:
Undo this. Keep the specific to issue/PR.

Reply:
Good call. We also added `throws InterruptedException` to the other PingPong tests.